scripts/release_candidate.py

```
@@ -3520,6 +3603,24 @@ def _validate_release_identity(
+        path,
+        label="effective upgrade-baseline policy",
+        max_bytes=MAX_EFFECTIVE_UPGRADE_BASELINES_BYTES,
+    )
+    _load_upgrade_baseline_policy(version, path)
+    after = _read_bounded_regular_file(
+        path,
+        label="effective upgrade-baseline policy",
+        max_bytes=MAX_EFFECTIVE_UPGRADE_BASELINES_BYTES,
+    )
+    if before != after:
+        raise CandidateError("effective upgrade-baseline policy changed during validation")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)